mkt/settings.py

```diff
@@ -1006,8 +1006,13 @@ def JINJA_CONFIG():
 READ_ONLY = False
 
 REDIS_BACKENDS = {
-    'master': os.environ.get('REDIS_URL',
```

Comment:
I don't remember what was the conclusion of the discussions on this ; don't you need to patch the settings in `sites/` too ?